components/plan-build-ps1/bin/hab-plan-build.ps1

```diff
@@ -340,8 +340,10 @@ function Install-Dependency($dependency, $install_args = $null) {
         $origin = $dependency.Split("/")[0]
         $channel = $env:HAB_BLDR_CHANNEL
         $ignoreLocal = ""
-        if ($origin -eq "core") {
-            $channel="$env:HAB_REFRESH_CHANNEL"
+        if ($origin -eq "core" -or $origin -eq "chef") {
+            if ($origin -eq "core") {
+                $channel = $env:HAB_REFRESH_CHANNEL
```

Comment:
This logic here and in the linux shell version is the one question I have about this PR. In thinking about it I'm sure you're correct but I'm still going to ask "why don't we want to set the HAB_REFRESH_CHANNEL for the chef origin?"